src/main.rs

```diff
@@ -93,6 +94,13 @@ fn print_file<P: AsRef<Path>>(
     line_changes: &Option<LineChanges>,
 ) -> Result<()> {
     let mut highlighter = HighlightFile::new(filename.as_ref(), syntax_set, theme)?;
+    if let Some(lang) = options.language {
+        let syntax = syntax_set.syntaxes().iter()
+            .find(|s| s.name.to_lowercase() == lang.to_lowercase());
```

Comment:
Is there a better way to compare strings by ignoring case? I couldn't find any, and this isn't the best because of the extra allocations

Reply:
There is [`eq_ignore_ascii_case`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.eq_ignore_ascii_case), which should be good enough for our purposes (we don't need Unicode support). We might need to bump the minimum-required Rust version for this.

Reply:
I don't know how I missed that. Thanks!